tests/phpunit/integration/tests/REST_API/Stories_Controller.php

```diff
@@ -141,6 +141,8 @@ public static function wpSetUpBeforeClass( $factory ): void {
 	public function set_up(): void {
 		parent::set_up();
 
+		$story_post_type = $this->injector->make( \Google\Web_Stories\Story_Post_Type::class );
+		$story_post_type->register();
```

Comment:
Why do we need to register the post type again now? Shouldn't it be registered already?

Reply:
Nope, rest apis are a special case and dont use the injector in the same. So this has to be done.